benchmark/fs/readfile-clogging.js

```diff
@@ -0,0 +1,74 @@
+// Call fs.readFile over and over again really fast.
+// Then see how many times it got called.
+// Also report how many times we were able to run a small concurrent
+// operation on the threadpool.
+'use strict';
+
+const path = require('path');
+const common = require('../common.js');
+const filename = path.resolve(__dirname, '.removeme-benchmark-garbage');
+const fs = require('fs');
+const zlib = require('zlib');
+
+const bench = common.createBenchmark(main, {
+  dur: [5],
+  len: [1024, 16 * 1024 * 1024],
+  concurrent: [1, 10]
+});
+
+function main(conf) {
+  const len = +conf.len;
+  try { fs.unlinkSync(filename); } catch (e) {}
+  var data = Buffer.alloc(len, 'x');
+  fs.writeFileSync(filename, data);
+  data = null;
+
+  var zipData = Buffer.alloc(1024, 'a');
+
+  var reads = 0;
+  var zips = 0;
+  var bench_ended = false;
+  bench.start();
+  setTimeout(function() {
+    console.log(`bench ended, reads ${reads} zips ${zips}`);
+    bench_ended = true;
+    bench.end(reads);
+    bench.end(zips);
```

Comment:
Calling this twice does not make sense and will break compare.js which is expecting one `bench.end()` per benchmark.

Reply:
@mscdex Thanks for pointing this out. How ought I report throughput for two separate variables like this?

Reply:
You can't. Perhaps just combine both values for *total* fulfilled requests per second?

Reply:
OK. I'll leave in the console.log then so the distinction between request type is clear.